ui/cypress/e2e/collectors.test.ts

```diff
@@ -369,5 +369,31 @@ describe('Collectors', () => {
           })
       })
     })
+    describe('Label creation and searching', () => {
+      beforeEach(() => {
+        const description = 'Config Description'
+        cy.get('@org').then(({id}: Organization) => {
+          cy.createTelegraf('newteleg', description, id, 'zoebucket')
+        })
+        cy.reload()
+      })
+      it('Can add label', () => {
+        cy.getByTestID('inline-labels--add').click()
+        cy.getByTestID('inline-labels--popover-field').type('zoe')
```

Comment:
i think that this label name should be changed to not share any letter with `zoebucket` (the name of your bucket above) so that we can have confidence we're not actually filtering the bucket name instead of the label.

Reply:
Agreed! fixed it!